app/views/questionnaire.py

```diff
@@ -40,21 +41,26 @@
                                     import_name=__name__,
                                     url_prefix='/questionnaire/<eq_id>/<form_type>/<collection_id>/')
 
+after_submission_blueprint = Blueprint(name='after_submission',
```

Comment:
after submission or post submission? just my opinion, ignore it if you like

Reply:
I'll toss a coin but I might weight it in your favour as I probably agree.